src/main/java/de/retest/recheck/ui/descriptors/AttributeUtil.java

```diff
@@ -1,9 +1,32 @@
 package de.retest.recheck.ui.descriptors;
 
 import java.awt.Rectangle;
+import java.util.List;
+
+import de.retest.recheck.ui.diff.AttributeDifference;
+import de.retest.recheck.ui.diff.ElementDifference;
+import de.retest.recheck.ui.diff.IdentifyingAttributesDifference;
 
 public class AttributeUtil {
 
+	public static Rectangle getActualOutline( final ElementDifference difference ) {
+		final Rectangle actualOutline = getActualOutline( difference, OutlineAttribute.RELATIVE_OUTLINE );
```

Comment:
Great refactoring, but I think this is a bit more readable:

```suggestion
		final Rectangle relative = getActualOutline( difference, OutlineAttribute.RELATIVE_OUTLINE );
```

The same applies to method name, i.e., something like `getActualRelativeOutline(…)`.